recipe/meta.yaml

```diff
@@ -40,3 +40,4 @@ about:
 extra:
   recipe-maintainers:
     - jzuhone
```

Comment:
@neutrinoceros I've talked to @ngoldbaum and he should also be added back in: `ngoldbaum`

Reply:
I'm very happy about that, and done !